webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -96,6 +96,29 @@ EnabledIcon.propTypes = {
   isOverridden: PropTypes.bool.isRequired,
 };
 
+const OsRestrictedIcon = ({ osRestricted }) => (
+  <Tooltip
+    position="right"
+    content={<FormattedMessage
+      id="os-restricted-tooltip"
+      defaultMessage={__('OS restricted to {osRestricted}')}
+      values={{ osRestricted }}
+    />}
+  >
+    <Label color="blue" className="os-restricted-label">
```

Comment:
The label is too close to the previous label. Recommend adding a style tag with a `marginLeft`